tests/Integration/Behaviour/Features/Context/Domain/StoreFeatureContext.php

```diff
@@ -58,4 +99,30 @@ public function isStoreToggledWithReference(string $reference, string $status):
         $storeUpdated = $this->getQueryBus()->handle($storeForEditingQuery);
         Assert::assertEquals((bool) $storeUpdated->isActive(), $isEnabled);
     }
+
+    /**
+     * @Then /^stores "(.+)" should be (enabled|disabled)$/
+     *
+     * @param string $storeReferences
+     * @param string $expectedStatus
+     */
+    public function assertMultipleStoreStatus(string $storeReferences, string $expectedStatus): void
+    {
+        foreach (PrimitiveUtils::castStringArrayIntoArray($storeReferences) as $storeReference) {
+            $this->assertStatus($storeReference, $expectedStatus);
+        }
+    }
+
+    public function assertStatus(string $storeReference, string $expectedStatus): void
```

Comment:
```suggestion
    public function assertSoreStatus(string $storeReference, string $expectedStatus): void
```

Reply:
you mean `assertStoreStatus`? :D

Reply:
Yes indeed, sorry for the typo :)

Reply:
Function name changed, no worries @saulaski ;)